server/internal/usecase/interactor/schema.go

```diff
@@ -150,6 +150,9 @@ func (i Schema) UpdateField(ctx context.Context, param interfaces.UpdateFieldPar
 				}
 				f.SetKey(k)
 			}
+			f.SetRequired(*param.Required)
+			f.SetUnique(*param.Unique)
+			f.SetMultiValue(*param.MultiValue)
```

Comment:
```suggestion
			if param.Required != nil {
				f.SetRequired(*param.Required)
			}

			if param.Unique != nil {
				f.SetUnique(*param.Unique)
			}

			if param.MultiValue != nil {
				f.SetMultiValue(*param.MultiValue)
			}
```

Have you checked that it works properly? It seems to me that some parameters cause panic.

Reply:
after a meeting with Basel we agreed to send the parameters again on update. however, those checks are safer. thank you